src/Discord.Net.WebSocket/ConnectionManager.cs

```diff
@@ -26,14 +28,14 @@ internal class ConnectionManager
         public ConnectionState State { get; private set; }
         public CancellationToken CancelToken { get; private set; }
 
-        internal ConnectionManager(SemaphoreSlim stateLock, Logger logger, int connectionTimeout, 
-            Func<Task> onConnecting, Func<Exception, Task> onDisconnecting, Action<Func<Exception, Task>> clientDisconnectHandler)
+        internal ConnectionManager(SemaphoreSlim stateLock, Logger logger, int connectionTimeout,
+            Func<Task> connect, Func<Exception, Task> disconnect, Action<Func<Exception, Task>> clientDisconnectHandler)
         {
             _stateLock = stateLock;
             _logger = logger;
             _connectionTimeout = connectionTimeout;
-            _onConnecting = onConnecting;
-            _onDisconnecting = onDisconnecting;
+            this._connect = connect;
```

Comment:
To be consistent with the conventions used in this library, `this` is not used, in favor of prefixing private fields with `_`. Ergo, the use of `this` is redundant.

Reply:
Wow really, I wouldn't have guessed...

(_I see that your superior treatment of others haven't changed. You should really try to be more friendly. I already commented about my editor being configured for completely different code formatting, you should have assumed that I just didn't notice that it used `this` notation instead of assuming that i'm a blind idiot.)_

Reply:
Statement of fact is not being mean.

Reply:
I'm merely passing on what I learned during my time managing and leading leading development teams at work for the benefit of the "manager" in this situation. Take it or leave it.

> Statement of fact is not being mean.

If left on its own in this situation, it is. Statement of a fact should be followed by your "story" that talks about the "why" and if it's this obvious that the person already knows your facts, either say that you're not trying to discourage them or use other technique to "soften it," or just don't state your fact in this manner in the first place - they already know it and you're wasting keyboard strokes.